staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -349,6 +349,13 @@ type ListOptions struct {
 	// add, update, and remove notifications. Specify resourceVersion.
 	// +optional
 	Watch bool `json:"watch,omitempty" protobuf:"varint,3,opt,name=watch"`
+	// AllowBookmarks opts-in for receiving watch events with type Bookmark.
```

Comment:
Use the camelCase name `allowBookmarks requests periodic watch events with type "BOOKMARK".`